### ISSUE TITLE
Vagrantfile should automatically git clone the three required repos

### ISSUE THREAD

**Issue by pacharanero** [RippleOSI/Ripple-Stack-Vagrant-Docker#3]
Currently the README.md requires the user to 

`git clone https://github.com/pacharanero/ripple-stack-vagrant-docker.git` # git clone this repository
`git clone https://github.com/PulseTile/PulseTile-React-Core.git ripple-pulsetile` Puts Pulsetile in ripple/ripple-pulsetile
`git clone https://github.com/RippleOSI/Ripple-QEWD-Microservices ripple-qewd` Puts QEWD in our ripple/ripple-qewd

in order to arrive at a file structure which looks like this:

ripple
    ├── ripple-pulsetile
    ├── ripple-qewd
    └── ripple-stack-vagrant-docker

These `git clone` steps should be automatically carried out by the Vagrantfile so that the user does not need to execute them

**Comment by robdyke** [comment]
I don't think that this should be included in the Vagrantfile - we are trying to avoid `git chaos` after all! There are other ways of achieving this though, perhaps with a Makefile.

**Comment by pacharanero** [comment]
@robdyke thanks for comments.

I don't mind really how it's done, just trying to minimise any manual steps to end up with a reporducible development environment, so happy to look into Makefile or other options.

**Comment by robdyke** [comment]
The documentation is clear that this repo is a helper to use along with the other repos.

Label `wontfix` and/or `broken elsewhere`
